tests/publiccloud/transfer_repos.pm

```diff
@@ -25,6 +25,12 @@ sub run {
 
     select_console 'tunnel-console';
 
+    # Shortcut for not doing updates (useful for speedup of verification runs)
+    if (get_var('PUBLIC_CLOUD_SKIP_DOWNLOAD') == 1) {
```

Comment:
because you actually skipping whole module same thing can be done in `main.pm` . Only good reason to do it within module is to have better visibility on tests page that module is skipped. This was your idea too ?

Reply:
Indeed, if it's just removed from the schedule I might suspect that something is wrong with openQA. If the test breaks off with a info note, I know that it is skipped because the `setting` is set.

The aim is to provide better visibility in the case this setting is used wrong